packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,169 @@
+
+Organizing communications via threading is an effective way to keep related messages grouped together. In healthcare apps, threads can include SMS chains, in-app chats, email exchanges, and more. These threads can constitute an [encounter](/docs/communications/async-encounters/async-encounters), which can be represented on `Communication.encounter`. The `Communication.encounter` element references a specific encounter that is closely related to the communication, or during which the communication took place. In addition, you can use the `Communication.inResponseTo` element. This element is a an array of references to `Communication` elements that preceded the current one and can be useful for tying all the messages in a threads together. Threading communications in this way allows for users to easily view all messages relevant to a single encounter.
+
+## Codesystems
+
+It is recommended to use LOINC or SNOMED codes to classify `Communication.category` and `Communication.topic`, but it is also possible to use custom coding internal to your practice.
+
+Below is an example of organizing a communication using all three of LOINC, SNOMED, and a custom code.
+
+```
+{
+  resourceType: "Communication",
+	//...
+	category: [
+	  // A category coded using LOINC
+		{
+			text: "Alert",
+			coding: [
+				{
+					code: "74018-3"
+					system: "https://fhir.loinc.org",
+				}
+			]
+		},
+	],
+	//...
+	topic: {
+	  // A topic coded using SNOMED
+		text: "High blood pressure",
+		coding: [
+			{
+				code: "38341003",
+				system: "http://snomed.info/sct",
+				display: "Hypertensive Disorder"
+			}
+		]
+	}
+}
+
+{
+	resourceType: "Communication",
+	//...
+	// An example of how an internal custom coding could be used for both category and topic
+	category: [
+		{
+			text: "appointment-reminder",
+			coding: [
+				{
+					code: "apt-rem",
+					system: "https://example-practice.org"
+				}
+			]
+		}
+	],
+	//...
+	topic: {
+		text: "annual-physical"
+		coding: [
+			{
+				code: "annu-phys",
+				system: "https://example-practice.org"
+			}
+		]
+	}
+}
+```
+
+## Search and GraphQL
+
+Effectively gropuing and filtering your communications makes it easier to search and query for specific data.
+
+For example, if you want to get all of the communications between the provider and a specific patient, you could use the following query. In this, we query for a patient by their id, then search all of the communications that reference them. From these communications, we get the text of the communication, the sender and time it was sent, and the receiver and time it was received. Preview this query in [GraphiQL](graphiql.medplum.com)
```

Comment:
You'll need to link to this specific query in graphiql - here's a short video on how to do it https://share.descript.com/view/4S6nwWDkylH

High level here is that is very important that these queries are correct and tested.  Samples should be able to be cut and pasted into the tool and run and produce the correct results https://share.descript.com/view/4S6nwWDkylH

Reply:
I got the first query working for me [here.](https://graphiql.medplum.com/?query=cXVlcnkgZ2V0Q29tbXVuaWNhdGlvbnNGb3JQYXRpZW50ewoJIyBTZWFyY2ggZm9yIGEgc3BlY2lmaWMgcGF0aWVudCBieSBpZAogIFBhdGllbnQoaWQ6ICJleGFtcGxlLXBhdGllbnQtaWQiKSB7CiAgICByZXNvdXJjZVR5cGUKICAgIGlkCgkJbmFtZSB7CgkJCWZhbWlseQoJCQlnaXZlbgoJCX0KCQkjIFNlYXJjaCBmb3IgY29tbXVuaWNhdGlvbnMgdGhhdCByZWZlcmVuY2UgdGhlIHBhdGllbnQKICAgIGNvbW11bmljYXRpb25zUmVjZWl2ZWQ6IENvbW11bmljYXRpb25MaXN0KF9yZWZlcmVuY2U6IHJlY2lwaWVudCkgewogICAgICB0ZXh0IHsKICAgICAgICBkaXYKICAgICAgfQogICAgICBzZW5kZXIgewogICAgICAgIHJlc291cmNlIHsKICAgICAgICAgIC4uLiBQcmFjdGl0aW9uZXJEZXRhaWxzCiAgICAgICAgICAuLi5QYXRpZW50RGV0YWlscwogICAgICAgIH0KICAgICAgfQogICAgICByZWNpcGllbnQgewogICAgICAgIHJlc291cmNlIHsKICAgICAgICAgIC4uLiBQYXRpZW50RGV0YWlscwogICAgICAgICAgLi4uUHJhY3RpdGlvbmVyRGV0YWlscwogICAgICAgIH0KICAgICAgfQogICAgfQogICAgY29tbXVuaWNhdGlvbnNTZW50OiBDb21tdW5pY2F0aW9uTGlzdChfcmVmZXJlbmNlOiBzZW5kZXIpIHsKICAgICAgdGV4dCB7CiAgICAgICAgZGl2CiAgICAgIH0KICAgICAgc2VuZGVyIHsKICAgICAgICByZXNvdXJjZSB7CiAgICAgICAgICAuLi4gUHJhY3RpdGlvbmVyRGV0YWlscwogICAgICAgICAgLi4uUGF0aWVudERldGFpbHMKICAgICAgICB9CiAgICAgIH0KICAgICAgcmVjaXBpZW50IHsKICAgICAgICByZXNvdXJjZSB7CiAgICAgICAgICAuLi4gUGF0aWVudERldGFpbHMKICAgICAgICAgIC4uLlByYWN0aXRpb25lckRldGFpbHMKICAgICAgICB9CiAgICAgIH0KICAgIH0KICB9Cn0KZnJhZ21lbnQgUHJhY3RpdGlvbmVyRGV0YWlscyBvbiBQcmFjdGl0aW9uZXIgewoJbmFtZSB7CgkJZmFtaWx5CgkJZ2l2ZW4KCX0KfQpmcmFnbWVudCBQYXRpZW50RGV0YWlscyBvbiBQYXRpZW50IHsKCW5hbWUgewoJCWZhbWlseQoJCWdpdmVuCgl9Cn0%3D&variables=&operationName=Z2V0Q29tbXVuaWNhdGlvbnNGb3JQYXRpZW50)
However, if I put "example-patient-id" it gives an error of "invalid input syntax." Should I just put a real id in so that it doesn't throw an error when the query is run?
The second one is pretty simple, so I should have that one ready soon as well.

Reply:
Yeah, recommend putting in a real patient ID